homeassistant/components/utility_meter/select.py

```diff
@@ -30,28 +30,10 @@ async def async_setup_entry(
 
     unique_id = config_entry.entry_id
 
-    registry = er.async_get(hass)
-    source_entity = registry.async_get(config_entry.options[CONF_SOURCE_SENSOR])
-    dev_reg = dr.async_get(hass)
-    # Resolve source entity device
-    if (
-        (source_entity is not None)
-        and (source_entity.device_id is not None)
-        and (
-            (
-                device := dev_reg.async_get(
-                    device_id=source_entity.device_id,
-                )
-            )
-            is not None
-        )
-    ):
-        device_info = DeviceInfo(
-            identifiers=device.identifiers,
-            connections=device.connections,
-        )
-    else:
-        device_info = None
+    device_info = d.async_device_info_to_link_entity(
```

Comment:
Let's import this method directly, we don't have to import devices as an alias in this case.